substrate/utils/wasm-builder/src/lib.rs

```diff
@@ -411,7 +380,7 @@ fn get_bool_environment_variable(name: &str) -> Option<bool> {
 
 /// Returns whether we need to also compile the standard library when compiling the runtime.
 fn build_std_required() -> bool {
-	let default = runtime_target() == RuntimeTarget::Wasm;
+	let default = RuntimeTarget::new() != None;
 
 	crate::get_bool_environment_variable(crate::WASM_BUILD_STD).unwrap_or(default)
```

Comment:
```suggestion
	crate::get_bool_environment_variable(crate::WASM_BUILD_STD).unwrap_or(true)
```

The default check here is pointless; if `runtime_target` returns `None` then the build will return an error and exit anyway, so effectively `default` is always going to be `true` here.